fixtures/interactive-dev-tests/tests/index.test.ts

```diff
@@ -292,3 +288,158 @@ it.each(exitKeys)("multiworker cleanly exits with $name", async ({ key }) => {
 		expect(duringProcesses.length).toBeGreaterThan(beginProcesses.length);
 	}
 });
+
+baseDescribe.skipIf(process.platform !== "linux" && process.env.CI === "true")(
+	"container dev",
+	{ retry: 0, timeout: 90000 },
+	() => {
+		let tmpDir: string;
+		beforeAll(async () => {
+			tmpDir = fs.mkdtempSync(path.join(tmpdir(), "wrangler-container-"));
+			fs.cpSync(
+				path.resolve(__dirname, "../..", "container-app"),
```

Comment:
could you copy the relevant files from the `container-app` here instead of adding a dependency from one fixture to another? this doesn't feel right to me 🤔